tools/add_triage_signature.py

```diff
@@ -1393,8 +1393,16 @@ def __init__(self, *args, **kwargs):
 
     def _process_ticket(self, url, issue_key):
         md = get_metadata_json(url)
-        cluster_id = md["cluster"]["id"]
+        cluster_hosts = md["cluster"]["hosts"]
+        bootstrap_node = [host for host in cluster_hosts if host["bootstrap"]][0]
+        eligible_bootstrap_stages = ["Rebooting", "Configuring", "Joined", "Done"]
```

Comment:
also ``Failed``, no?

Reply:
Nope, this is progress stage, not state

Reply:
Bootstrap progress stages are:
         "Starting installation",
          "Installing",
          "Writing image to disk",
          "Waiting for control plane",
          "Waiting for bootkube",
          "Waiting for controller",
          "Rebooting",
          "Configuring",
          "Joined",
          "Done"